infra-templates/k8s/12/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ kubelet:
         - --register-node=true
```

Comment:
@imikushin could you create a new folder for this change? as we've already released 12. Use:

*  v1.4.6-rancher2 tag for k8s images
* put minimum rancher version as 1.3
* release an official image for kubedns-starter. Do we have rancher repo for it? if not, we should create it. 
* call service as addon-starter. As potentially it can start more than one service (dashboard, helm, etc)